tests/ffi/sysctl/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+exec_cmd "podman run -d --rm --replace -d --name ffi-asil \
+          quay.io/centos-sig-automotive/ffi-tools:latest  > /dev/null"
+
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
+if [ ! -d "${QM_HOST_REGISTRY_DIR}" ]; then
+    exec_cmd "mkdir -p ${QM_HOST_REGISTRY_DIR}"
+    exec_cmd "podman push ${image_id} dir:${QM_HOST_REGISTRY_DIR}/ffi-tools:latest"
+fi
+
+podman exec -it qm /bin/bash -c \
+         "podman run  --replace --name ffi-qm  dir:${QM_REGISTRY_DIR}/ffi-tools:latest \
+         ./setsysctl > /dev/null"
+
+if [ $? -eq 137 ]; then
```

Comment:
okay, ffi-qm was killed by SIGKILL, is it expected?  I didn't follow. 
Also, I would recommend not using "magic numbers" if possible. Please use some constants to describe the numbers, example:

# Signal number for SIGKILL
SIGNAL=9
# Base exit code for signal-based terminations
EXIT_CODE_BASE=128
# Calculate the expected exit code for SIGKILL
SIGKILL_CODE=$((EXIT_CODE_BASE + SIGNAL))

In the end: 128+9=137

Reply:
Just to add: I completely understand what you tried to do using directly 137 and it's fine. What I am trying to share here is that in 5 or 7 years someone might read this code and do not understand what we tried to do here. :) Thanks!

Reply:
Looking at logs,
I see many 

```
13:05:04                 out: Relabeled /var/lib/containers from unconfined_u:object_r:unlabeled_t:s0 to system_u:object_r:container_var_lib_t:s0
```

After test is called

Reply:
I assume that you need to catch this error
`sysctl: permission denied on key `

as per #370 
and this https://github.com/containers/qm/tree/main/tests/e2e/tools/FFI/sysctl

Reply:
I look this again, I think it is unnecessary. I'll remove it.